xml/installation-installation-configure_3par.xml

```diff
@@ -125,29 +125,23 @@
  <section xml:id="sec.3par-multipath">
   <title>Multipath Support</title>
   <para>
-   If you want to enable multipath for Cinder volumes carved from 3PAR FC/iSCSI
-   storage system, please go through the
+   Setting up multipath support is highly recommended for 3PAR FC/iSCSI
+   backends, and should be considered a default best practice.
+   For instructions on this process, refer to the
    <filename>~/openstack/ardana/ansible/roles/multipath/README.md</filename>
    file on the &clm;. The <filename>README.md</filename> file contains
    detailed procedures for configuring multipath for 3PAR FC/iSCSI Cinder
    volumes.
   </para>
+  <warning>
+   <para>
+    If multipath functionality is enabled, ensure that all 3PAR fibre channel ports are active and zoned correctly in the 3PAR storage for proper operation.
+   </para>
+  </warning>
   <para>
-   We have also included an additional set of steps needed if you are using
-   3PAR FC/iSCSI multipath which is included below.
+   In addition, take the following steps to enable 3PAR FC/iSCSI multipath
+   support in &ostack; configuration files:
```

Comment:
"...support in the &ostack; configuration files:"

Reply:
You could also probably remove "In addition"

Reply:
Not really, it needs to be very clearly a follow-on step associated with the previous clause.

Reply:
But I can use simpler wording and just change it to "also"